contrib/terraform/openstack/README.md

```diff
@@ -85,6 +87,28 @@ number_of_k8s_nodes = "0"
 ```
 This will provision one VM as master using a floating ip, two additional masters using no floating ips (these will only have private ips inside your tenancy) and one VM as node, again without a floating ip.
 
+##### Alternative: etcd on separate machines
+
+If you want to provision master or node VMs that don't use floating ips and where **etcd is not in the same machines as masters**, write on a `my-terraform-vars.tfvars` file, for example:
+
+```
+number_of_etcd = "3"
+number_of_k8s_masters = "0"
```

Comment:
Is there an option to do etcd+master+node here? I just want to understand the options correctly

Reply:
Yes, the original option `k8s_master`, set by `number_of_k8s_masters > 0` will produce VMs that will be each tagged in ansible for etcd, kube-master, kube-node, k8s-cluster and vault simultaneously, and they will use each a floating ip. In the particular example that you are citing, you wouldn't get VMs tagged like that. Let me know if it needs further clarification.